app/assets/javascripts/application.js

```diff
@@ -6,7 +6,6 @@
 //= require scoped_search
 //= require bootstrap
 //= require patternfly
-//= require two-pane
```

Comment:
this change should be reverted, as it breaks other pages/plugins that would need to require it themselves.